### DISCUSSION TITLE
Specify multiple startup profiles

### DISCUSSION THREAD

**Discussion by BitEater21** [Eugeny/tabby#4903] [General]
I would like to be able to specify (via config file or via command line) a list of specific profiles to open at startup in a specific order

**Comment by composer404** [comment]
Hey, I wrote a plugin that provides a possibility to create workspace profiles with different tabs configuration (you can set a title, color, profile of a terminal and starting commands), maybe this will be helpful for you.

There is also a possibility to select a particular profile on a terminal startup and assign first five to hotkeys.

https://github.com/composer404/tabby-workspace-manager

**Reply by BitEater21** [reply]
Thank you for the hint. It seems I missed this when having a look on the plugins. 
It would be great if it would be possible to specify the profiles to load not by index but by a short ID, because index order may change. Perhaps the basic tabby configuration could be extended by such an ID?!

**Reply by BitEater21** [reply]
One question about the Workspace manager: There is an option to run a workspace profile at startup. If not activated - how can I activate a workspace profile at runtime? Or even if activated - it is possible to create multiple workspace profiles - how can I switch?

**Reply by composer404** [reply]
I posted this plugin yesterday, so you may not have seen it on the list :)

As for the selection of profiles, I have already received one request, so I created an issue in the plugin's repository.

I don't quite understand the second question. At the moment, in the settings tab, you can select and enable the workspace profle to be launched at startup (upper right corner). Additionally, in the hotkeys tab, you can assign the first five profiles workspace.

**Reply by BitEater21** [reply]
@composer404: Today I added a new profile. tabby added this as first in the config file. This totally messed up the IDs for the Workspace manager. Now the IDs are not related to their order in the config file. I am a bit confused...

**Reply by composer404** [reply]
In a new version of the workspace manager, instead of id, you can use profile's name
